.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: java
 jdk: oraclejdk8
 dist: trusty
-
-# Make the gradle wrapper executable
-before_install: chmod +x gradlew
+sudo: false
+script:
+  - bash gradlew check
+cache:
```

Comment:
Why not also add the before_cache lines from https://docs.travis-ci.com/user/languages/java/#caching ?